src/contracts/test/GnosisSafeERC1271Fallback.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity >=0.5.0 <0.7.0;
```

Comment:
Just curious as why this version range was used instead of just:
```solidity
pragma solidity ^0.5.17;
```

(not requesting that this be changed, just curious if there was a reason).

Reply:
Because this is the version range specified in the Safe contracts. I picked Solidity `0.5.17` for the compiler because this is what the deployed master copy on mainnet uses.